addon.py

```diff
@@ -30,7 +30,7 @@ def index():
     try:
         classes = loadClasses(username, password)
     except requests.HTTPError as ex:
-        if ex.code != 401:
+        if not hasattr(ex, 'code') or ex.code != 401:
```

Comment:
I was getting exceptions when trying to run this on an older version of python. That version didn't have the correct version of the Requests lib. I was only able to determine this after making these changes.